Tests/ApiEndpoints/test_allhousingapp_pytest.py

```diff
@@ -47,3 +47,36 @@ def test_put_apartment_application(self):
         if not response.status_code == 200:
             pytest.fail('Expected 200 Created, got {0}.'\
                 .format(response.status_code))
+
+#    Verify that a user who is on the admin whitelist gets the OK to access staff features
+#    Endpoint -- 'api/housing/admin'
+#    Expected Status Code -- 200 OK
+#    Expected Content -- Empty response content
+    def test_is_on_whitelist(self):
+        self.session = self.createAuthorizedSession(control.username, control.password)
+        self.url = control.hostURL + 'api/housing/admin'
+        # add test user to whitelist
+        api.post(self.session, self.url + control.my_id_number + '/')
+        # check that user is on the whitelist
+        response = api.get(self.session, self.url)
+        # remove 
+        api.delete(self.session, self.url + control.my_id_number + '/')
```

Comment:
```suggestion
        api.delete(self.session, self.url + '/' + control.my_id_number + '/')
```
Same suggestions here regarding the '/'